tools/IceRpc.Slice.Tools/IceRpc.Slice.Tools.csproj

```diff
@@ -42,7 +42,7 @@
    <ItemGroup>
       <PackageReference Include="Microsoft.Build.Framework" Version="17.8.3" />
       <PackageReference Include="Microsoft.Build.Utilities.Core" Version="17.8.3" />
-      <PackageReference Include="System.Text.Json" Version="8.0.*" />
```

Comment:
Would be nicer to fix this issue in a separate PR.

Reply:
Yes, already did